lib/matplotlib/backends/backend_wx.py

```diff
@@ -1236,6 +1236,7 @@ def new_figure_manager_given_figure(num, figure):
     figmgr = frame.get_figure_manager()
     if matplotlib.is_interactive():
         figmgr.frame.Show()
+        figure.canvas.draw_idle()
```

Comment:
I seem to recall that draw_idle() in wx can be broken sometimes. Would it make sense to make all of these be `figure.canvas.draw()` instead as this isn't a performance-critical portion of the code? We should probably guarantee an image upon initial display.